src/index.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-classes-per-file */
+import './style.css';
+
+const btn = document.getElementById('add-btn');
+
+class Item {
+  constructor(text) {
+    this.text = text;
+    this.completed = false;
+  }
+}
+
+class List {
+  static dataArray() {
+    const listArray = JSON.parse(localStorage.getItem('listArray') || '[]');
+    return listArray;
```

Comment:
* Kindly add some default tasks which would be rendered on page load currently, i have to add tasks before it is rendered [Refer back to project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=On%20page%20load%20render%20the%20dynamically%20created%20list%20of%20tasks%20in%20the%20dedicated%20placeholder.%20The%20list%20should%20apear%20in%20order%20of%20the%20index%20values%20for%20each%20task.)